.github/workflows/test-commit.yml

```diff
@@ -0,0 +1,41 @@
+name: test commit
+on: [pull_request]
+jobs:
+  test-classifier:
+    name: Run classifier tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Run format test
+        run: cd opentutor_classifier && make test-format
+      - name: Run lint test
+        run: cd opentutor_classifier && make test-lint
+      - name: Run types test
+        run: cd opentutor_classifier && make test-types
+      - name: Run license test
+        run: cd opentutor_classifier && make test-license
+      - name: Run tests
+        run: cd opentutor_classifier && make test
+  test-api:
+    name: Run api tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Run format test
+        run: cd opentutor_classifier_api && make test-format
+      - name: Run lint test
+        run: cd opentutor_classifier_api && make test-lint
+      - name: Run types test
+        run: cd opentutor_classifier_api && make test-types
+      - name: Run license test
+        run: cd opentutor_classifier_api && make test-license
+      - name: Run tests
+        run: cd opentutor_classifier_api && make test
```

Comment:
these 3 new yaml files still missing the the newline at the EOF

Reply:
I see them in the code though? I think they just don't show in github.